.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+### Description
+
+#### What is changing?
+
+##### Is there new documentation needed for these changes?
+
+#### What is the motivation for this change?
+
+<!-- If this is a bug, it helps to describe the current behavior and a clear outline of the expected behavior -->
+<!-- If this is a feature, it helps to describe the new use case enabled by this change -->
+
+<!--
+Contributors!
+First of all, thank you so much!!
+If you haven't already, it would greatly help the team review this work in a timely manner if you create a JIRA ticket to track this PR.
+You can do that here: https://jira.mongodb.org/projects/NODE
+-->
+
+### Release Highlight
+
+<!-- RELEASE_HIGHLIGHT_START -->
+
+### Fill in title or leave empty for no highlight
+
+<!-- RELEASE_HIGHLIGHT_END -->
+
+### Double check the following
+
```

Comment:
Do we need the release highlight section here? We don't currently have any packages that we release in this repository and we also don't have the release automation set up for this repository.

Reply:
It could one day! but fair enough removed.